src/index.html

```diff
@@ -1,13 +1,443 @@
+                grid__item
+                grid__item--tablet--2-5
+                grid__item--desktop--5-8
+              "
+            >
+              <div class="card abuot-us__card">
+                <div class="card__img-container">
+                  <img
+                    src="./images/photos/SoundLink Flex Bluetooth speaker.jpg"
+                    alt="SoundLink Flex Bluetooth speaker"
+                    class="card__product-img"
+                  >
+                </div>
+                <h3 class="card__title">
+                  Bose portable Smart speaker
+                </h3>
+                <p class="card__description">
+                  Smart home
+                </p>
+                <p class="card__price">
+                  $ 399.00
+                </p>
+              </div>
+            </div>
+            <div class="
+                grid__item
+                grid__item--tablet--2-5
+                grid__item--desktop--9-12
+              "
+            >
+              <div class="card abuot-us__card">
+                <div class="card__img-container">
+                  <img
+                    src="./images/photos/SoundLink Color Bluetooth speaker II.jpg"
+                    alt="SoundLink Color Bluetooth speaker II"
+                    class="card__product-img"
+                  >
+                </div>
+                <h3 class="card__title">
+                  Bose portable Smart speaker
+                </h3>
+                <p class="card__description">
+                  Smart home
+                </p>
+                <p class="card__price">
+                  $ 399.00
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="now-on-view page__section" id="now-on-view">
+        <div class="container">
+          <h2 class="page__section-title">
+            Browse Bose
+            <br>
+            products by category
+          </h2>
+          <div class="place now-on-view__place">
+            <div class="place__photos">
+              <div class="grid">
+                <div class="
+                    grid__item
+                    grid__item--tablet--1-4
+                    grid__item--desktop--1-8
+                  "
+                >
+                  <div class="place__photo-container">
+                    <img
+                      src="./images/photos/1.jpg"
+                      alt="product photo 1"
+                      class="place__photo"
+                    >
+                  </div>
+                </div>
+                <div class="
+                    grid__item
+                    grid__item--tablet--5-6
+                    grid__item--desktop--9-12
+                  "
+                >
+                  <div class="place__photo-container">
+                    <img
+                      src="./images/photos/2.jpg"
+                      alt="product photo 2"
```

Comment:
add more describable alt, it means not just 'product photo 1/2/3/4' -> 'woman with headphones and camera'